javascript_nodejs/10.prompt-validations/index.js

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const restify = require('restify');
+const path = require('path');
+const ERROR = 1;
```

Comment:
Please change to `const CONFIG_ERROR = 1;` [per this comment](https://github.com/Microsoft/BotBuilder-Samples/pull/291#issuecomment-417427551).